### PR TITLE
add TDD for get request on /api/topics endpoint

### PR DESCRIPTION
**Pull request by HudaifDaif**

### CODE REVIEW

File: api/errors.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+exports.handleBadPath = (req, res, next) => {
+	res.status(400).send({ msg: "Bad Request" });
+};
```
Comment (Keb84) [inline api/errors.js:3]: this status code to handle an incorrect path should be a 404 👀

**Review by Keb84 — CHANGES_REQUESTED** [review]
Please check my comments and make the change 🙂

File: api/errors.js (hunk 1 above)
Comment (HudaifDaif) [reply api/errors.js:3]: Will do, thank you. I've been putting 400 for this all the time haha

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the change Daif, all good to merge! 🙌

**Pull request merged**